### PR TITLE
Resolve #653, Resolve #674 -- Don't copy Contents

### LINKED ISSUE #653: Fix LeagueSandbox-Default not updating in bin folder
A quick fix would be to make it read from a path in a config file rather than from the bin folder. Should save a lot of time copying files too.

---
Beep, boop, I'm [a bot](https://github.com/LeagueSandbox/IssueBot)! This issue was created by `@Gan` in `#random🍝`.

### LINKED ISSUE #674: Improve exception throwing in GameServerLib Program.cs in the StartNetworkLoop
It doesn't preserve the stack trace in the debug window. It only logs the stack trace. Makes it difficult to look at variables.

---
Beep, boop, I'm [a bot](https://github.com/LeagueSandbox/IssueBot)! This issue was created by `@Gan` in `#general`.

### PR DESCRIPTION
**Pull request by FurkanS1821**
Contents folder is no longer copied
Refs #653
Debugging is made easier right now by removing main try loop for debug
builds.
Refs #674

### CODE REVIEW

File: GameServerLib/Config.cs [hunk 1]
```diff
@@ -69,12 +69,15 @@ private void LoadConfig(Game game, string json)
             // Read where the content is
             ContentPath = (string)gameInfo.SelectToken("CONTENT_PATH");
 
+            // Load items
+            game.ItemManager.LoadItems(ContentPath);
```
Comment (MythicManiac) [inline GameServerLib/Config.cs:73]: What's up with this addition?

Comment (FurkanS1821) [reply GameServerLib/Config.cs:73]: It was moved from somewhere upper in the execution order to there to prevent issues...
I think.

Comment (MythicManiac) [reply GameServerLib/Config.cs:73]: Yeah looks like it

**Review by MythicManiac — APPROVED** [review]
LGTM 👍

**Pull request merged**